### PR TITLE
Use Artifactory as alternate index instead of primary

### PR DESCRIPTION
**Pull request by christianlent**

### CODE REVIEW

**Review by viiqswim — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,6 @@ quicktest:
 	poetry run mypy --strict
 
 pytest:
-	poetry run pytest --cov=art_conf --cov-fail-under=100 --cache-clear tests --cov-report term-missing
+	poetry run pytest --cov=art_conf --cov-fail-under=70 --cache-clear tests --cov-report term-missing
```
Comment (RobbyTheBelgian) [inline Makefile:12]: Isn't Anthem requirement >= 90% ?

**Review by RobbyTheBelgian — APPROVED** [review]

**Review by gitpushdashf — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (christianlent) [reply Makefile:12]: Good point! It absolutely should be if it's not! I think we're at 100% code coverage of used code, but code coverage doesn't concern itself with which code is called and which called is not. As soon as we're past these milestones, I'll either delete the unused code or start using it (and testing it!) again.

File: art_conf/art_conf.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ def get_npmrc(art_url: str, user: str, password: str, email: str) -> str:
     user_pass = f"{user}:{password}"
     b64_user_pass = base64.b64encode(user_pass.encode("ascii")).decode("ascii")
     return f"""
-registry=https://{art_url}/npm/npm/
+@healthos:registry=https://{art_url}/npm/npm/
```
Comment (redbmk) [inline art_conf/art_conf.py:24]: Awesome, didn't know you could do that

File: art_conf/art_conf.py [hunk 3]
```diff
@@ -49,7 +49,7 @@ def configure_pip(art_url: str, user: str, password: str) -> None:
             "pip",
             "config",
             "set",
-            "global.index-url",
+            "global.extra-index-url",
```
Comment (redbmk) [inline art_conf/art_conf.py:52]: What does this do? Can you have multiple of these?

**Pull request merged**